api/src/test/java/org/openmrs/api/impl/ConceptServiceImplTest.java

```diff
@@ -828,4 +828,22 @@ public void saveConceptProposal_shouldReturnUpdatedConceptProposalObject() {
 	public void saveConceptProposal_shouldFailGivenNull() {
 		conceptService.saveConceptProposal(null);
 	}
+
+	/**
+	 * @see ConceptServiceImpl#getConceptNameTagByName(String)
+	 */
+	@Test
+	public void getConceptNameTagByName_shouldReturnNullIfNoConceptNameTagIsFound() {
+		assertNull(conceptService.getConceptNameTagByName("random-tag"));
+	}
+
+	/**
+	 * @see ConceptServiceImpl#getConceptNameTagByName(String)
+	 */
+	@Test
+	public void getConceptNameTagByName_shouldReturnTheMatchingConceptNameTagObjectIfFound() {
+		ConceptNameTag c = conceptService.getConceptNameTag(1);
```

Comment:
Don't name the variables without proper conventions. Better to rename this `c` to some meaning full name

Reply:
Would `conceptNameTag` be fine?

Reply:
Looking at naming conventions, I've renamed it :)